monitoring/snippets/v3/uptime-check-client/snippets.py

```diff
@@ -83,26 +123,44 @@ def update_uptime_check_config(
     if new_http_check_path:
         field_mask.paths.append("http_check.path")
         config.http_check.path = new_http_check_path
-    client.update_uptime_check_config(request={"uptime_check_config": config, "update_mask": field_mask})
+    changed_config = client.update_uptime_check_config(
+        request={"uptime_check_config": config, "update_mask": field_mask}
+    )
+    pprint.pprint(changed_config)
+    return changed_config
 
 
 # [END monitoring_uptime_check_update]
 
 
 # [START monitoring_uptime_check_list_configs]
-def list_uptime_check_configs(project_name):
+def list_uptime_check_configs(project_id: str) -> str:
+    """Gets all uptime checks defined in the Google Cloud project
+
+    Args:
+        project_id: Google Cloud project id
+
+    Returns:
+        A list of uptime check configurations
+    """
     client = monitoring_v3.UptimeCheckServiceClient()
-    configs = client.list_uptime_check_configs(request={"parent": project_name})
+    configs = client.list_uptime_check_configs(request={"parent": project_id})
 
     for config in configs:
         pprint.pprint(config)
+    return configs
 
 
 # [END monitoring_uptime_check_list_configs]
 
 
 # [START monitoring_uptime_check_list_ips]
-def list_uptime_check_ips():
+def list_uptime_check_ips() -> None:
```

Comment:
It's actually returning a List of ip objects.

Reply:
fixed